Sorting/merge.cpp

```diff
@@ -5,6 +5,72 @@
  *
  * Inputs: null
  */
-void Sort::MergeSort(){
-    std::cout<<"[TODO @ Sorting/merge.cpp] MergeSort() needs to be implemented.\n";
-}
+std::vector<int> arr;
```

Comment:
This should be avoided. Merge sort is a function from a class. It's okay to use a function from another scope(namespace or such) but using a global array which is not a part of that class should be avoided.

Reply:
@yatendra1999  Done.